modules/nf-core/getorganelle/config/tests/main.nf.test

```diff
@@ -0,0 +1,73 @@
+import groovy.io.FileType
+
+nextflow_process {
+
+    name "Test Process GETORGANELLE_CONFIG"
+    script "../main.nf"
+    process "GETORGANELLE_CONFIG"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "getorganelle"
+    tag "getorganelle/config"
+
+    test("test-getorganelle-config") {
+
+        when {
+            process {
+                """
+                input[0] = 'all'
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                {
```

Comment:
Why is this so complicated?

Reply:
It is because of the nested folders. The emitted folder contains subfolders which contain files. Using the simple `files(xyz).listFiles()` is not an option.